debian/ceph-mgr-ssh.install

```diff
@@ -0,0 +1 @@
+usr/lib/ceph/mgr/ssh
```

Comment:
s/lib/share/

Reply:
oh! thanks kefu. i realize I did change to `datadir`